migrations/1563351138480-addStartAndEndDateToYearCollection.js

```diff
@@ -32,6 +32,6 @@ module.exports = {
 	},
 
 	down: async function down() {
-		throw new Error('down is not supported for this migration');
+		warning('down is not supported for this migration');
```

Comment:
Bitte aus dem PR rausziehen. Das hat nichts mit dem Fix zu tun

Reply:
In my opinion, throwing a error at this point is a bug. We can't roll back the migrations.